reference/forms/types/range.rst

```diff
@@ -0,0 +1,56 @@
+.. index::
+   single: Forms; Fields; range
+
+range Field Type
+================
+
+The ``range`` field is a slider that is rendered using the HTML5
+``<input type="range" />`` tag.
+
++-------------+---------------------------------------------------------------------+
+| Rendered as | ``input`` ``range`` field (slider in html5 supported browser)                              |
```

Comment:
Basically the editor was having `Soft wrap at preferred length` turned on . I changed it. Thanks.